Makefile

```diff
@@ -86,7 +85,8 @@ fmt:
 
 lint:
 	@echo "+ $@"
-	@test -z "$$(golint ./... | grep -v .pb. | grep -v Godeps/_workspace/src/ | tee /dev/stderr)"
+	# TODO: remove trustmanager/yubikey/ from ignored once linted
```

Comment:
Oh nm well idk I have a bad feeling it's going to cause something bad eventually